docs/jotai/guides/atoms-in-atom.mdx

```diff
@@ -138,7 +138,7 @@ const pricesAtom = atom({
 })
 
 const Fruit = ({ name, priceAtom }) => {
-  const [price] = useAtom(priceAtom)
+  const [price] = useAtom(countAtom)
```

Comment:
This is reverting a fix. `priceAtom` is correct one.

Reply:
I've restored the fix.